### DISCUSSION TITLE
Meaning of parameters in collapse boundary model

### DISCUSSION THREAD

**Discussion by zaizibai** [lnccbrown/HSSM#401] [General]
Dear HSSM developers,

First of all, thanks for making this wonderful package. I am using the weibull collapse boundary in HSSM. Weibull collapse boundary model has been widely used since Hawkins et al., 2015 introduced it. In the Hawkins et al., 2015 paper, the boundary collapse like the following way:
![image](https://github.com/lnccbrown/HSSM/assets/44595163/0ac32a05-71ed-40e8-aa86-75f995823748)
The parameter lambda is a shape parameter that determines the extent to which the boundary collapses, the parameter k determines when the boundary starts to collapse, and a' is the asymptote of the collapsed boundary.

However, in the weibull collapse boundary model in HSSM, there are only two parameters for the collapsed boundary: alpha and beta. What is the meaning of these two parameters. And why one parameter is missing? Thanks a lot.

Best,
MQ Guo

**Comment by frankmj** [comment]
Glad that you are enjoying HSSM!

The weibull bound as implemented in hssm is described in the 2021 LAN paper
<https://elifesciences.org/articles/65074> -- B(t) = a*exp((-t^alpha)/beta)
where a is the initial height, beta is the scale (related to the onset) of
the collapse, and alpha is the shape. This is the standard weibull
distribution. So compared to Hawkins, alpha = k and beta = lambda.

Hawkins also added an additional parameter to allow the asymptotic value of
the bound to be a' whereas in our case the two bounds eventually meet to
force a decision. But this is still pretty flexible (with sufficiently high
alpha or beta the bounds will not meet for a long time). And as is, we
found that there can be some identifiability issues with recovering alpha
from beta in some cases depending on how much of the data lie within the
range of the bound trajectory (see the paper above for discussion).  Note
that Hawkins and others usually fixed at least one of the parameters
(including setting a' to 0) when fitting the model rather than estimating
all of them.

That said, one can certainly train a new LAN to also allow for an
asymptotic bound and add this to the hssm bank. I would just suggest that
it would be important to do parameter recovery for the experimental design
you are using.

Michael

**Reply by zaizibai** [reply]
Hi Michael, thanks for your detailed explanation!
